src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/DetailsSection/DurationFields.tsx

```diff
@@ -0,0 +1,55 @@
+import { Trans } from '@lingui/macro'
+import TooltipLabel from 'components/TooltipLabel'
+import { durationOptions } from 'components/inputs/DurationInput'
+import { JuiceSwitch } from 'components/inputs/JuiceSwitch'
+import { CYCLE_EXPLANATION } from 'components/strings'
+import DurationInputAndSelect from 'components/v2v3/shared/FundingCycleConfigurationDrawers/FundingDrawer/FundingForm/DurationInputAndSelect'
+import { useEffect, useState } from 'react'
+import { useEditCycleForm } from '../EditCycleFormContext'
+
+export function DurationFields() {
+  const [durationEnabled, setDurationEnabled] = useState<boolean>()
+
+  const { editCycleForm, initialFormData } = useEditCycleForm()
+
+  // Set initial value of Switch
+  useEffect(() => {
```

Comment:
avoid useEffect for state changes if you can - very error prone and creates unintentional bugs

Reply:
Yep this was unnecessary, worked the initial value into the state definition now and it's working fine